modulemd/v2/include/modulemd-2.0/modulemd-module.h

```diff
@@ -33,6 +33,18 @@ G_DECLARE_FINAL_TYPE (
   ModulemdModule, modulemd_module, MODULEMD, MODULE, GObject)
 
 
+/**
+ * modulemd_module_index_get_stream_names_as_strv: (rename-to modulemd_module_index_get_stream_names)
```

Comment:
You use the `(rename-to)` directive here, but in order to do that you also need to update
https://github.com/fedora-modularity/libmodulemd/blob/master/modulemd/v2/include/private/gi-binding-renames.h